### ISSUE TITLE
Do you know why just one small change (**1 line**) greatly improves accuracy `+ 0.7` on Pytorch-yolo?

### ISSUE THREAD

**Issue by AlexeyAB** [WongKinYiu/CrossStagePartialNetworks#9]
@WongKinYiu Hi,

Do you know why just one small change (**1 line**) greatly improves accuracy `+ 0.7` on Pytorch-yolo? https://github.com/WongKinYiu/CrossStagePartialNetworks/blob/pytorch/README.md

| Model | Size | NMS | 1080ti *fps* | BFLOPs | AP  | AP50 | AP75 | cfg | weight |
| :---- | :--: | :-: | :----------: | :----: | :--: | :--: | :--: | :-: | :----: |
| **CSPResNeXt50-PANet-SPP** | 512×512 | 0.5 | 44 | 71.331 | 39.2 | 59.5 | 41.8 | [cfg](https://github.com/WongKinYiu/CrossStagePartialNetworks/blob/pytorch/cfg/csresnext50-panet-spp.cfg) | - |
| **CSPResNeXt50c-PANet-SPP** | 512×512 | 0.5 | - | 71.734 | 39.9 | 60.1 | 42.6 | [cfg](https://github.com/WongKinYiu/CrossStagePartialNetworks/blob/pytorch/cfg/csresnext50c-panet-spp.cfg) | - |

![image](https://user-images.githubusercontent.com/4096485/72907899-c37cdf80-3d45-11ea-9f27-9a5070b681b5.png)

**Comment by WongKinYiu** [comment]
@AlexeyAB 

Hello, 273 epochs is not sufficient for training from scratch of panet.
From the log file, I think it not yet converge.

In rethinking imagenet pre-training, they also shows that training form scratch need more epochs.
![image](https://user-images.githubusercontent.com/12152972/72958516-7ac73400-3de2-11ea-9228-45a059d5583a.png)

**Comment by AlexeyAB** [comment]
So +-0.7 AP can be just fluctuation on early stages of training.
Or partial-residual connections require more iterations for training than common-residual connections?

**Comment by WongKinYiu** [comment]
I think both yes.
